### PR TITLE
 Fix the Ballerina.toml updating before the Ballerina push

### PR DESCRIPTION
**Pull request by SasinduDilshara**
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Comment by BuddhiWathsala** [comment]
@SasinduDilshara, it's better if you can add the Github action as here(https://github.com/ballerina-platform/module-ballerina-time/pull/100/files). That action will help to publish artifacts to the Ballerina central manually if something goes wrong in the release process.

**Comment by SasinduDilshara** [comment]
> @SasinduDilshara, it's better if you can add the Github action as here(https://github.com/ballerina-platform/module-ballerina-time/pull/100/files). That action will help to publish artifacts to the Ballerina central manually if something goes wrong in the release process.

I will add

File: .github/workflows/central-publish.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            -   name: Grant execute permission for gradlew
+                run: chmod +x gradlew
+            -   name: Publish artifact
+                env:
+                    BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_ACCESS_TOKEN }}
+                    packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+                    packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                    GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                run: |
+                    ./gradlew clean build -PpublishToCentral=true
```
Comment (BuddhiWathsala) [inline .github/workflows/central-publish.yml:25]:
```suggestion
                    ./gradlew clean build -PpublishToCentral=true
                    
```

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

**Review by BuddhiWathsala — APPROVED** [review]
LGTM

**Pull request merged**